.github/workflows/ci.yml

```
@@ -67,19 +63,68 @@ jobs:
       - name: Test package
         run: python -m pytest -vv -rs -Wd
 
+  checks_alt:
+    name: Check Python ${{ matrix.python-version }}, ${{ matrix.arch }} on ${{ matrix.runs-on }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.11"]
+        runs-on: [ubuntu-latest]
+        arch: [aarch64]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: uraimo/run-on-arch-action@v2.5.0
+        name: Install and test in image
+        with:
+          arch: ${{ matrix.arch }}
+          distro: ubuntu_latest
+          # Not required, but speeds up builds
+          githubToken: ${{ github.token }}
+          dockerRunArgs: |
+            --volume "${PWD}:/fastjet" --platform linux/arm64
+          install: |
+            uname -m
+            apt-get update
+            apt-get install -y ca-certificates gnupg2 build-essential patch
+            echo 'deb https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy main' >> /etc/apt/sources.list
+            echo 'deb-src https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy main' >> /etc/apt/sources.list
+            apt-key adv --keyserver keyserver.ubuntu.com --recv-keys BA6932366A755776
+            apt-get update
+            apt-get install -y libboost-dev libmpfr-dev swig autoconf libtool python${{ matrix.python-version }} python${{ matrix.python-version }}-dev python${{ matrix.python-version }}-distutils python${{ matrix.python-version }}-venv
```

```suggestion
            apt-get install -y libboost-dev libmpfr-dev swig autoconf libtool python${{ matrix.python-version }} python3-dev python${{ matrix.python-version }}-distutils python${{ matrix.python-version }}-venv
```

Do you actually need `python3.7-dev`/`python3.11-dev`? Or can you just use `python3-dev` here?

---

That will install all of python3.10 as well because of deps. I don't want to get into that kind of dependency hell.